### ISSUE TITLE
[BUG] Gray modal blinking during authentication process

### ISSUE THREAD

**Issue by iankressin** [spruceid/ssx#90]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

After SXX `signIn` method is called, a gray modal flashes in the screen and before the `/ssx-nonce` http call.
Due to how fast the modal show us and disappear on the screen, I wasn't able to screenshot it.

As I was trying to screen shot the modal, I tried to apply network throttling, and notice that on really slow connections the modal doesn't show up.

The network throttling specs used:
Download: 100 kbit/s
Upload: 10 kbit/s
Latency: 10000 ms

It also doesn't show up when a `debugger` is placed right before `signIn` method being called, and we walk through every step of the debugger.

### Expected Behavior

Not displaying any UI element during the sign in process

### Steps To Reproduce

There's only one step that's calling the sign-in method. On my app, I'm doing as follows:

```ts
import { PUBLIC_EXODA_API } from '$env/static/public'
import { SSX } from "@spruceid/ssx";

/**
* Interacts with Exoda's API to handle authentication process
*/
export class AuthService {
    /**
    * Uses SSX to create a session between 
    *
    * @return user address or throws
    */
    public static async auth(): Promise<string> {
        const { address } = await AuthService._ssx.signIn()

        if (!address)
            throw new Error('AuthService::auth: address no present on response from the server')

        return address
    }

    private static get _ssx(): SSX {
        return new SSX({
            enableDaoLogin: true,
            resolveEns: true,
            providers: {
                web3: {
                    driver: window.ethereum,
                },
                server: {
                    host: PUBLIC_EXODA_API,
                    routes: {
                        nonce: '/nonce',
                        login: '/login',
                        logout: '/logout',
                    }
                },
            },
        })
    }
}

```

**Comment by Juliano1612** [comment]
Hi @iankressin, thank you for your feedback. 

This gray modal may probably be related to the DAO Login - which if no delegation is found automatically selects your address and closes the modal. We'll check into verifying the delegations beforehand.

About the network issue, I couldn't reproduce it. Can you open a new issue with more information about it?

We can also schedule a conversation if you want to.

**Comment by iankressin** [comment]
hey @Juliano1612, you're right about the DAO login. I set `enableDaoLogin` to `false` and the modal doesn't show up anymore.

About the network issue: I'm not quite sure what's the issue there, nor the expected behavior. I was trying to make the modal to stay a little longer on the screen so I could screenshot it, when I notice that on slow connections the gray modal never shows up.

I'm available for a call if you need more details about anything

**Comment by obstropolos** [comment]
Updated with #98  will close when updated in release

**Comment by obstropolos** [comment]
This was addressed with [ssx-gnosis-extension/v1.1.5](https://github.com/spruceid/ssx/releases/tag/ssx-gnosis-extension-v1.1.5)
